DigitalLearningSolutions.Web/Styles/trackingSystem/delegates/delegateGroups.scss

```diff
@@ -0,0 +1,12 @@
+﻿@import 'delegateCommon';
+
+.nhsuk-details__text > .nhsuk-button {

```

Comment:
I'm slightly scared by this because the HTML file will look like we're just using nhsuk's styles, and future devs probably wouldn't expect the button's styling to change just because they move it, or because they add a div between it and the `.nhsuk-details__text`.

I think a better approach would be to put an extra class on the buttons you're selecting here and select them as `.nhsuk-button.<new-class>` - that gets around the problems above, and may even get rid of the need for `!important`.

Reply:
Yep, that is definitely better and fixes the specificity issue I was having resulting in this selector and all the !important declarations.

Reply:
Excellent! Maybe useful for the name of the new class to mention `nhsuk-button` or somehow make clear it should be used in conjunction with it and won't do anything if you put it on any old button, if that can be done without making it annoyingly lanky, but no big deal.